src/twisted/conch/ssh/keys.py

```diff
@@ -1563,75 +1487,24 @@ def _toPrivateOpenSSH_PEM(self, passphrase=None):
         @param passphrase: The passphrase to encrypt the key with, or L{None}
         if it is not encrypted.
         """
-        if self.type() == "EC":
-            # EC keys has complex ASN.1 structure hence we do this this way.
-            if not passphrase:
-                # unencrypted private key
-                encryptor = serialization.NoEncryption()
-            else:
-                encryptor = serialization.BestAvailableEncryption(passphrase)
-
+        if not passphrase:
+            # unencrypted private key
+            encryptor = serialization.NoEncryption()
+        else:
+            encryptor = serialization.BestAvailableEncryption(passphrase)
+        if self.type() != "Ed25519":
             return self._keyObject.private_bytes(
                 serialization.Encoding.PEM,
                 serialization.PrivateFormat.TraditionalOpenSSL,
                 encryptor,
             )
-        elif self.type() == "Ed25519":
+        else:
+            # TODO: why not just support serialization here
```

Comment:
@alex @reaperhulk I'm going to merge this as-is since I don't want to block "CI working at all" on this minor TODO detail, but I'd really love an issue that followed up with this and removed the TODO to put it in a more actionable place (i.e. an issue that explains the serialization issue)